code/modules/mob/living/simple_mob/subtypes/animal/alien animals/teppi.dm

```diff
@@ -0,0 +1,1195 @@
+	skin_color = baby.skin_color
+	ghostjoin = 1
+	active_ghost_pods |= src
+	update_icon()
+
+//This sets all the things on baby teppi when they are bred from adult teppi
+/mob/living/simple_mob/vore/alienanimals/teppi/proc/inherit_from_parents(mob/living/simple_mob/vore/alienanimals/teppi/mom, mob/living/simple_mob/vore/alienanimals/teppi/dad)
+	inherit_colors = TRUE
+//	mom_id = mom.teppi_id
+//	dad_id = dad.teppi_id
+	faction = mom.faction
+	color = pick(list(mom.color, dad.color, BlendRGB(mom.color, dad.color, 0.5)))
+	marking_color = pick(list(mom.marking_color, dad.marking_color, BlendRGB(mom.marking_color, dad.marking_color, 0.5)))
+	horn_color = pick(list(mom.horn_color, dad.horn_color, BlendRGB(mom.horn_color, dad.horn_color, 0.5)))
+	eye_color =  pick(list(mom.eye_color, dad.eye_color, BlendRGB(mom.eye_color, dad.eye_color, 0.5)))
+	skin_color =  pick(list(mom.skin_color, dad.skin_color, BlendRGB(mom.skin_color, dad.skin_color, 0.5)))
+	marking_type =  pick(list(mom.marking_type, dad.marking_type, null))
+	horn_type =  pick(list(mom.horn_type, dad.horn_type, null))
+
+
+	if(mom.teppi_mutate || dad.teppi_mutate)
+		teppi_mutate = TRUE
+	else if(prob(1))
+		teppi_mutate = TRUE
+	mom.nutrition -= 500
+	dad.nutrition -= 250
+	mom.visible_message("\The [src] is born from [mom]... It's the miracle of life!", runemessage = "grunts")
+	handle_affinity(mom, 26)	//this way the babies will follow their parents around (and keep track of them)
+	handle_affinity(dad, 25)
+
+//I ran a vote with the headmins, and this option won out considering the restrictions.
+//I don't think this is a GOOD idea, but in pursuit of preserving Teppi's mechanical functionality while player controlled, there is a verb!
+//This gives a strongly worded warning the first time you push the button, and has similar restrictons to AI controlled Teppi for use which will prevent spamming.
+//
+/mob/living/simple_mob/vore/alienanimals/teppi/proc/produce_offspring()
+	set name = "Produce Offspring"
+	set category = "Abilities"
+	set desc = "You can have babies if the conditions are right."
+	if(prevent_breeding)
+		to_chat(src, "<span class='notice'>You have elected to not participate in breeding mechanics, and so cannot complete that action.</span>")
+		return
+	if(!teppi_warned)
+		to_chat(src, "<span class='danger'>Be aware of your surroundings when using this verb. If you use this to be disruptive or prefbreak people, you are likely to eat a ban. If whoever's tending the teppi is trying to make more babies, or you're alone, or playing with other people who you know are into it, then sure. You should not however, for example, drag another teppi to the bar (or any public place) and drop a baby in the middle of the floor. If you're not sure if it's okay to do where you are, with whoever's around, it probably isn't. This is intended to preserve the mechanical utility of the mob you are playing as, not as a scene tool.</span>")
+		teppi_warned = TRUE
+		return
```

Comment:
Comment: Doesn't account for multiple people playing the same teppi over time, but that's probably pedantic. I handled that on the translocator beacons but those are much more likely to be touched by more than one person, so eh!

Reply:
This is true, though, I'm not super sure how to resolve that... hmm. Maybe Login should set teppi_warned to false?

Reply:
I wouldn't worry too much about it, honestly.